.github/workflows/codebuild-ci.yml

```diff
@@ -139,6 +139,20 @@ jobs:
         uses: aws-actions/aws-codebuild-run-build@v1
         with:
           project-name: aws-sdk-java-v2-endpoints-test
+  brazil-json-validation:
+    if: github.repository == 'aws/aws-sdk-java-v2'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.CI_AWS_ROLE_ARN }}
+          aws-region: us-west-2
+          role-duration-seconds: 7200
+      - name: Validate Brazil config
+        uses: aws-actions/aws-codebuild-run-build@v1
+        with:
+          project-name: aws-java-sdk-v2-validate-brazil-config
```

Comment:
`aws-java-sdk` or `aws-sdk-java` ? this file uses both 😭

Reply:
I just copied one of the existing jobs, I don't think we've been very consistent unfortunately!